### DISCUSSION TITLE
Unblock Allowing Permission for Remote Assistance (Anydesk)

### DISCUSSION THREAD

**Discussion by rubisco57** [simeononsecurity/Windows-Optimize-Harden-Debloat#53] [Q&A] [answered]
Great script, thanks a bunch! 
How do I unblock popular remote assistance tools, e.g. Anydesk, for a normal user account though? I could not find the appropriate settings; currently a pop-up advises "This app has been blocked by your system administrator".

**Comment by simeononsecurity** [comment, marked as answer]
> Great script, thanks a bunch! How do I unblock popular remote assistance tools, e.g. Anydesk, for a normal user account though? I could not find the appropriate settings; currently a pop-up advises "This app has been blocked by your system administrator".

@rubisco57 
I'm going to assume by "normal" you mean unprivileged. This is a test case we haven't considered in the past as on non-enterprise systems being an administrator is the default. However, nothing in this repo should be preventing users from executing any application unless it's picked up by windows defender. Also Applocker and WDAC are both set to audit only on this repo so it shouldn't be flagging anything except for logging purposes. Though, for testing we suggest you attempt the following:
1. Open gpedit.msc
2. Go to Local Computer Policy > Computer Configuration > Windows Settings > Security Settings > Application Control Policies > Applocker 
3. Delete any of the existing rules if any
4. Go to the following locations on your "C" drive. "C:\Windows\System32\CodeIntegrity\CiPolicies\Active" and delete all the policies there
5. Go to "C:\Windows\System32\CodeIntegrity" and delete the .p7b policies and reboot

If the application is still blocked after this, it isn't because of the script in this repo.

**Reply by simeononsecurity** [reply]
We should say that disabling WDAC policies isn't exactly straight forward and how we've suggested it here isn't the cleanest solution. 
It might be smarter to just delete WDAC policies before running the script inside of the folder structure for future runs. https://github.com/simeononsecurity/Windows-Optimize-Harden-Debloat/tree/master/Files/Windows%20Defender%20Configuration%20Files
